metrics/network/network-metrics-iperf3.sh

```diff
@@ -75,10 +94,29 @@ function iperf3_bidirectional_bandwidth_client_server {
 	local client_command="mount -t ramfs -o size=20M ramfs /tmp && iperf3 -c ${server_address} -d -t ${time}"
 	start_client "$extra_args" "$client_name" "$image" "$client_command" > "$result"
 
-	local total_bidirectional_client_bandwidth=$(cat $result | tail -n 3 | head -1 | awk '{print $(NF-2), $(NF-1)}')
-	local total_bidirectional_server_bandwidth=$(cat $result | tail -n 4 | head -1 | awk '{print $(NF-3), $(NF-2)}')
-	echo "Network bidirectional bandwidth (client to server) is : $total_bidirectional_client_bandwidth"
-	echo "Network bidirectional bandwidth (server to client) is : $total_bidirectional_server_bandwidth"
+	local client_result=$(grep -m1 -E '\breceiver\b' $result)
+	local server_result=$(grep -m1 -E '\bsender\b'   $result)
+	local -a client_results
+	read -a client_results <<< ${client_result}
+	read -a server_results <<< ${server_result}
+	local total_bidirectional_client_bandwidth=${client_results[6]}
+	local total_bidirectional_client_bandwidth_units=${client_results[7]}
+	local total_bidirectional_server_bandwidth=${server_results[6]}
+	local total_bidirectional_server_bandwidth_units=${server_results[7]}
+	echo "Network bidirectional bandwidth (client to server) is :" \
+		"$total_bidirectional_client_bandwidth" \
+		"$total_bidirectional_client_bandwidth_units"
+	echo "Network bidirectional bandwidth (server to client) is :" \
+		"$total_bidirectional_server_bandwidth" \
+		"$total_bidirectional_server_bandwidth_units"
+
+	save_results "${test_name}" "network bidir bw client to server" \
```

Comment:
If it makes more sense to put each result in an independent file, that's fine, and pretty easy to accomplish. I'd just take the unique info out of Args and append it to Test Name. Let me know if you'd prefer that, or adding another field, etc...